### PR TITLE
New language Files.zh-Hant.xlf

### PR DESCRIPTION
**Pull request by cracky5322**
Added Traditional Chinese Language

### CODE REVIEW

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 1]
```diff
@@ -0,0 +1,808 @@
...
+          <source>Desktop</source>
+          <target state="translated">桌面</target>
+        </trans-unit>
+        <trans-unit id="SidebarDocuments" translate="yes" xml:space="preserve">
+          <source>Documents</source>
+          <target state="translated">文件</target>
+        </trans-unit>
+        <trans-unit id="SidebarDownloads" translate="yes" xml:space="preserve">
+          <source>Downloads</source>
+          <target state="translated">下载</target>
```
Comment (NoobTW) [inline Files/MultilingualResources/Files.zh-Hant.xlf:211]: 下載

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 2]
```diff
@@ -0,0 +1,808 @@
...
+          <source>Videos</source>
+          <target state="translated">影片</target>
+        </trans-unit>
+        <trans-unit id="NavigationToolbarNewTab.Text" translate="yes" xml:space="preserve">
+          <source>New Tab</source>
+          <target state="translated">新增頁籤</target>
+        </trans-unit>
+        <trans-unit id="ExtractFilesDialog.Title" translate="yes" xml:space="preserve">
+          <source>Extract Compressed Archive</source>
+          <target state="translated">解壓所檔案</target>
```
Comment (NoobTW) [inline Files/MultilingualResources/Files.zh-Hant.xlf:235]: 解壓縮

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 3]
```diff
@@ -0,0 +1,808 @@
...
+          <source>Cancel</source>
+          <target state="translated">取消</target>
+        </trans-unit>
+        <trans-unit id="ExtractFilesDialog.PrimaryButtonText" translate="yes" xml:space="preserve">
+          <source>Extract</source>
+          <target state="translated">解壓縮</target>
+        </trans-unit>
+        <trans-unit id="ExtractFilesDialogDescription.Text" translate="yes" xml:space="preserve">
+          <source>Pick a location to extract this compressed archive to. You'll need to stay in the current folder until we're done. A new tab will open up with the extracted items.</source>
+          <target state="translated">選擇一个位置以解壓縮此壓縮檔。解壓所完成前請不要離開此頁，完成後將開啟新頁籤顯示解壓縮的檔案。</target>
```
Comment (NoobTW) [inline Files/MultilingualResources/Files.zh-Hant.xlf:247]: 解壓縮

**Comment by yaichenbaum** [comment]
@cracky5322 Can you make sure this new file is included in the cproj file? Otherwise it will not be be included in the application when we package it.

**Comment by cracky5322** [comment]
> @ cracky5322您可以確保此新文件包含在cproj文件中嗎？否則，當我們打包它時，它將不會包含在應用程序中。

**Comment by cracky5322** [comment]
> @cracky5322 Can you make sure this new file is included in the cproj file? Otherwise it will not be be included in the application when we package it.

Added Files.zh-Hant.xlf to Files.csproj

**Pull request merged**